### PR TITLE
Redesigned the Event card

### LINKED ISSUE #113: feat: Redesign Event Card
### **Is your feature request related to a problem? Please describe.**
The current event card component needs a redesign to improve visual appeal and functionality.

### **Describe the solution you'd like**
- Redesign the event card component to align with the theme defined in `theme.tsx`.
- Ensure the component is fully responsive to different screen sizes.
- Utilize the [MUI card component](https://mui.com/material-ui/react-card/).
- Use `MUI` and `CSS`
- The new UI must look as close as possible to the Figma design.

### **Additional context**
- Below is a screenshot of the desired event card UI for reference.
image

- The exact design can be found in this [figma design](https://www.figma.com/design/vDVEoB5YY32EUoeFWtXaCA/Lupo-Skill?node-id=14-174&t=ygGyLPPd7qlvcPlU-1).
- This screen recording shows how get exact colors and values from a figma design.

https://github.com/Tanay-ErrorCode/lupo-skill/assets/116624667/c1fe8e14-50af-46b6-b473-5dbeecbe3748

### PR DESCRIPTION
**Pull request by Utsavladia**
**What kind of change does this PR introduce?**

Redesigned the Event card according to the design provided in Figma.

**Issue Number:**

Fixes #113 
**Snapshots/Videos:**

Screenshot 2024-06-01 144316

**Other information**

### CODE REVIEW

**Comment by Tanay-ErrorCode** [comment]
@Utsavladia Please share a full screen video of the functionality.

**Comment by Utsavladia** [comment]
Here is the illustration of redesigned card.
Uploading cardillustration recording (1).mp4…

**Comment by Utsavladia** [comment]
Here is the video of redesigned Event card.

https://github.com/Tanay-ErrorCode/lupo-skill/assets/124615886/97a3f8b7-954a-40f4-bba6-9c0d08946163

File: src/components/Cards/EventCard/EventCard.tsx
Comment (Doraemon012) [inline src/components/Cards/EventCard/EventCard.tsx:207]: Update this using `theme.tsx`. Remember that never use `px` for font sizes, user `rem` instead. This helps keeping the webpage responsive.

File: src/components/Cards/EventCard/EventCard.tsx
Comment (Doraemon012) [inline src/components/Cards/EventCard/EventCard.tsx:194]: Use the font size `textBody` frm `theme.tsx`.

**Comment by Doraemon012** [comment]
@Utsavladia 
1. The description should cover max 3 lines, if it is more than that then it should be truncated by `...` Like this:
image

2. The heights of all cards needs to be same.
3. Remove `Indian Standard Time` from the event time.

File: src/components/Cards/EventCard/EventCard.tsx
Comment (Doraemon012) [inline src/components/Cards/EventCard/EventCard.tsx:350]: Use the [MUI Chip Component](https://mui.com/material-ui/react-chip/#clickable)

**Review by Doraemon012 — CHANGES_REQUESTED** [review]
Please make the requested changes

**Comment by Tanay-ErrorCode** [comment]
@Utsavladia kindly fix  height of the cards are not equal.

image

**Comment by Utsavladia** [comment]
Sure, Thank you for the review I will try to fix all...

**Comment by Utsavladia** [comment]
Here is the video after suggested changes


https://github.com/Tanay-ErrorCode/lupo-skill/assets/124615886/b0ea6a49-b9ee-4619-9377-031a9a4db5a6

**Comment by Doraemon012** [comment]
@Utsavladia The same card design should be applied for Dashboard and User Profile.

File: src/components/Cards/EventCard/EventCard.tsx
Comment (Doraemon012) [inline src/components/Cards/EventCard/EventCard.tsx:359]: Make this clickable. [https://mui.com/material-ui/react-chip/#clickable](https://mui.com/material-ui/react-chip/#clickable)

**Comment by Doraemon012** [comment]
- Dashboard cards should be clickable.
- The card title should be max 2 lines, if more than that then it should be truncated by `...`

File: src/components/Cards/EventCard/EventCard.tsx
Comment (Doraemon012) [inline src/components/Cards/EventCard/EventCard.tsx:362]: Make the font bold

File: src/components/Cards/EventCard/EventCard.tsx
Comment (Doraemon012) [inline src/components/Cards/EventCard/EventCard.tsx:217]: make the padding `2px 12px`

**Comment by Utsavladia** [comment]
Done all the suggested changes

Profile cards
image

Dashboard cards
image

clickable links and fixed padding.
image

**Comment by Utsavladia** [comment]
@Doraemon012  
I have resolved the conflicts and please let me know if changes required.
And kindly merge this after review else conflicts will keep coming...

**Comment by Tanay-ErrorCode** [comment]
@Utsavladia Please provide a video of the output. And please do not resolve merge conflicts on your own.

**Comment by Utsavladia** [comment]
sure @Tanay-ErrorCode 
Please find the video of output below..


https://github.com/Tanay-ErrorCode/lupo-skill/assets/124615886/e357c957-f974-46d7-bb2f-9005b2c19678

**Comment by Tanay-ErrorCode** [comment]
@Utsavladia Is this the video of the [latest commit ](https://github.com/Tanay-ErrorCode/lupo-skill/pull/158/commits/573af8424e70f4083d5e894b8c4e7fbf8956dac5)?

**Comment by Utsavladia** [comment]
@Tanay-ErrorCode 
yes this is the video of latest commit before merging the main or resolving the conflicts.

**Comment by Tanay-ErrorCode** [comment]
> @Tanay-ErrorCode yes this is the video of latest commit before merging the main or resolving the conflicts.

@Utsavladia I want the video after resolving merge conflicts. Please pull the latest commit changes.

**Comment by Utsavladia** [comment]
@Tanay-ErrorCode 
You are right i should not have resolved the conflict without asking.
It gave too many errors.
I have reverted to the last working commit that i mad.
The video above I attached is the video of that last commit.

**Review by Doraemon012 — APPROVED** [review]

**Pull request merged**
